### PR TITLE
Add proto file

### PR DESCRIPTION
**Pull request by 914-Mereu-Bianca**
- [x] add proto file
- [x] add data setting tests
- [x] add service test

### CODE REVIEW

File: tests/test_client.cc [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#include <iostream>
```
Comment (CristianIonutIonescu) [inline tests/test_client.cc:1]: this file has nothing to do with RTOPServer, if anything, should be in RTopClient repository

File: tests/test_server.cc [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#include <iostream>
```
Comment (CristianIonutIonescu) [inline tests/test_server.cc:1]: I'm failing to see any tests written here.. looks like what the actual implementation should be. Is this code in the correct place?

File: tests/test_server.cc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+using grpc::ServerBuilder;
+using grpc::ServerContext;
+using grpc::Status;
+using data::RTopServer;
+using data::RequestTopPackage;
+using data::ProcessInfo;
+using data::TopPackage;
+using data::GeneralInfo;
+using data::CpuUsage;
+using data::MemoryUsage;
```
Comment (CristianIonutIonescu) [inline tests/test_server.cc:17]: I would prefer to not make use of using directive globally. If need be, try to use it locally

**Review by CristianIonutIonescu — CHANGES_REQUESTED** [review]
Protofile looks good, some changes are needed on tests.
PS: try to use .cpp and .h instead of .cc/.hh

File: tests/test_server.cc (hunk 2 above)
Comment (914-Mereu-Bianca) [reply tests/test_server.cc:1]: For testing the service from the proto file I created a client and a server to manually test the communication between them. Should I try another approach?

Comment (CristianIonutIonescu) [reply tests/test_server.cc:1]: all unit test should run automatically, without any human intervention. Manual testing is a different thing.

**Review by enachebogdan — APPROVED** [review]

**Review by CristianIonutIonescu — APPROVED** [review]

**Pull request merged**